src/commands.rs

```diff
@@ -342,11 +339,6 @@ fn decompress_file(
     //
     // Any other Zip decompression done can take up the whole RAM and freeze ouch.
     if formats.len() == 1 && *formats[0].compression_formats == [Zip] {
-        if !utils::clear_path(output_dir, question_policy)? {
-            // User doesn't want to overwrite
-            return Ok(());
-        }
-        utils::create_dir_if_non_existent(output_dir)?;
```

Comment:
Why we won't clear path anymore?

Reply:
I hoisted that out of the function here https://github.com/ouch-org/ouch/pull/206/files/85968ea8410ea86b8a636aaacf1ff4280c3e0a84#diff-b86d43fce6a9c57160aa925719550d40a6ed3f5e5a0117679905f76e68acc520R209

There is a choice to make, what I went with here is if the directory exists we will just put the files in it without overwriting

The second choice is to ask for overwrite, but in that case we should *only* ask for overwrite if the user specified a dir with --dir (if he did not, the directory is "." and we should not ask to overwrite it)

Either way is fine, I just went with the 1st one because its less code

Reply:
I restored the previous behaviour

Reply:
> Either way is fine, I just went with the 1st one because its less code

I guess I agree, I'm just a bit worried if this will change other stuff or add some bugs, because we had some new bugs recently in `ouch` because I was short on time and ended up merging a lot of PRs without testing everything.

Reply:
Yes I definitely want to add more tests, at least everyone will feel "safer" when adding prs

Reply:
I don't think it's strictly necessary, majority of bugs I catch by running `ouch` two or four times but would be impossible to catch with tests, because they come in new features.